Robot_Engine/Convert/FromRobot/RigidLink.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
```

Comment:
Realised now as well, this method is not used anywhere, right? If so, I think we should just delete this file.